Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -158,8 +170,11 @@ extension BrowserViewController: TabBarViewControllerDelegate {
 
 extension BrowserViewController: TabManagerDelegate {
     func didSelectedTabChange(selected: Browser?, previous: Browser?) {
-        previous?.webView.hidden = true
-        selected?.webView.hidden = false
+        // previous?.webView.hidden = true
+        // selected?.webView.hidden = false
```

Comment:
Remove comment. Hopefully iOS is better than Android in that it doesn't try to draw views that are behind!